charts/datadog/templates/_container-agent.yaml

```
@@ -164,6 +164,10 @@
       mountPath: /var/log/pods
       mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
       readOnly: true
+    - name: logsymlinkspath
```

to follow the naming convention with `logspodpath` 

```suggestion
    - name: logscontainerspath
```